api/models/app.go

```diff
@@ -59,7 +59,32 @@ func ListApps() (Apps, error) {
 }
 
 func GetApp(name string) (*App, error) {
-	res, err := DescribeStack(name)
+	return GetAppFast(name, true, true)
+}
+
+func GetAppFast(name string, bound bool, unbound bool) (*App, error) {
```

Comment:
My first draft had some 2 consts in the file for use as flags, but that looked like overkill to me, so I removed them. The only real motivation for the 2 separate definitions is so most code can continue using `GetApp` but `(App|Service)Create` -- specifically -- can use `GetAppFast` to avoid cache misses and force query the unbound app first (since bound is done first now). LMK if changes are needed... even to terminology (I'm not sure I care for the un/bound terms, but it works well enough).